css/projects.css

```diff
@@ -0,0 +1,264 @@
+/* General CSS */
+
+body  {
+background-color: #E6DACD; 
+text-align:left;
+margin:0px;
+overflow-x: hidden;
+}
+
+a:link {
+    text-decoration: none;
+}
+
+h1{
+    font-weight:700;
+    font-size:2.5em;
+    padding-top: 3em;
+}
+
+h3 {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    font-size: 1.5em;
+    padding-left:5px;
+}
+
+/* CSS for Top Navigation */
+
+.hidex{
+    display:none;
+}
+
+.hamburger{
+    width:35px;
+    height:35px;
+    position:absolute;
+    top: 6px;
+    right: 6px;
+}
+
+
+.wrapsquare{
+    display:flex;
+    margin: 10px 0px 0px 0px;
+}
+
+.square {
+  height: 13.5px;
+  width: 13.5px;
+  background-color:#0150FD;
+  margin:9px 0px;
+}
+
+.developer {
+    font-weight:400;
+    font-size: 0.95em;
+    line-height: 2.2;
+}
+
+.toptext{
+    text-align:left;
+}
+
+
+
+header a{
+    font-size:.7em;
+}
+
+
+/* CSS for Profile Section */
+.jobposition{
+    font-weight:700;
+}
+.date{
+    color:#0150FD;
+    padding-top:3em;
+    font-weight:700;
+}
```

Comment:
Could you add something here to re-create the vertical blue bar from the design? 🙂